src/Cli/dotnet/Commands/Test/Terminal/ExceptionFlattener.cs

```diff
@@ -27,7 +25,7 @@ internal static FlatException[] Flatten(string? errorMessage, Exception? excepti
         IEnumerable<Exception?> aggregateExceptions = exception switch
         {
             AggregateException aggregate => aggregate.Flatten().InnerExceptions,
-            _ => [exception?.InnerException],
+            _ => [exception?.InnerException!],
```

Comment:
This is known issue in collection expression null inference that I needed to work around. @cston